lib/datasource/helm/common.ts

```diff
@@ -11,30 +11,46 @@ const githubRelease = regEx(
 
 export function findSourceUrl(release: HelmRelease): RepoSource {
   // it's a github release :)
-  let match = githubRelease.exec(release.urls[0]);
-  if (match) {
-    return { sourceUrl: match[1] };
+  const releaseMatch = githubRelease.exec(release.urls[0]);
```

Comment:
Please create tests for all cases, your changes are looking somehow wrong to me.

Reply:
Sure, I will. Is there something specific that looks wrong?